mono/mini/debugger-agent.c

```diff
@@ -11711,13 +11729,54 @@ string_commands (int command, guint8 *p, guint8 *end, Buffer *buf)
 	return ERR_NONE;
 }
 
+static void 
+create_file_to_check_memory_address (void) 
```

Comment:
Seems like some things could be behind a `_MSC_VER` check but are not.

Reply:
You're right this is only used in the `#ifndef _MSC_VER` check. I'll wrap this.